src/amo/components/HeroRecommendation/index.js

```diff
@@ -134,14 +161,32 @@ export class HeroRecommendationBase extends React.Component<InternalProps> {
             )}
             <div className="HeroRecommendation-info">
               <div className="HeroRecommendation-recommended">
-                {recommended}
+                {/* translators: If uppercase does not work in your locale, change it */}
```

Comment:
I believe this needs to be directly above the `i18n.gettext()` call. Can you do that with a double slash inside the ternary?